### PR TITLE
Feat Match Button with Design Guidelines

### PR DESCRIPTION
**Pull request by JoseRFelix**

### CODE REVIEW

**Review by jonator — CHANGES_REQUESTED** [review]
Please address merge conflicts

File: packages/web/components/assets/chevron-down-icon.tsx [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import classNames from "classnames";
+import { FunctionComponent } from "react";
+
+export const ChevronDownIcon: FunctionComponent<{
```
Comment (jonator) [inline packages/web/components/assets/chevron-down-icon.tsx:4]: Are you intending for this to replace all uses of `chevron-down.svg` (and it's variants)? If so, I think we should replace them all. Though, it's worth considering this approach will result in a slowly increased main bundle size.

Comment (JoseRFelix) [reply packages/web/components/assets/chevron-down-icon.tsx:4]: After some deliberation, I found this [article](https://benadam.me/thoughts/react-svg-sprites/?utm_source=pocket_reader). It posits that using a sprite SVG is a way to have the bundle benefits of the image while having some of the flexibility of inlining. I believe we could use this for our non-animated SVGs that only require shallow CSS properties. My latest commit introduces an Icon component that leverages this technique. I think this is a nice solution to this problem. What do you think?

File: packages/web/components/buttons/button.tsx [hunk 2]
```diff
@@ -1,59 +1,172 @@
...
+          "border-wosmongton-300",
+          "hover:border-wosmongton-200",
+          "text-wosmongton-300",
+          "hover:text-wosmongton-200",
+          "rounded-md",
+          "disabled:border-osmoverse-600",
+          "disabled:text-osmoverse-400",
+        ],
+        amount: [
+          "border-2",
```
[the remark this answers is not in the document]
Comment (JoseRFelix) [reply packages/web/components/buttons/button.tsx:83]: You're right! Fixed 👍

**Comment by JoseRFelix** [comment]
@jonator Replaced all chevron icons images with the new sprite method. I believe other icons can be migrated incrementally.

**Review by jonator — APPROVED** [review]

**Pull request merged**